### ISSUE TITLE
NTR: X-ray contrast enema: megacolon (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8974]
New term request X-ray contrast enema: megacolon (segal_130731201849): Large colon; barium  UMLS: 0025160

**Comment by pnrobinson** [comment]
@MickeySegal
is this distinct from [Aganglionic megacolon HP:0002251](https://hpo.jax.org/app/browse/term/HP:0002251) and if so how?

**Comment by MickeySegal** [comment]
Some aganglionic:  https://ojrd.biomedcentral.com/articles/10.1186/1750-1172-6-29 "Intestinal anomalies such as omphalocele, Hirschsprung disease, congenital aganglionic megacolon, stenosis of the small intestine, and inguinal hernia, renal, cardiac and vertebral anomalies, and cleft hands have been reported"

Some the opposite:  https://www.ncbi.nlm.nih.gov/books/NBK1257/ "About 40% of affected individuals have diffuse ganglioneuromatosis of the gastrointestinal tract. Associated symptoms include abdominal distention, megacolon, constipation, or diarrhea."

**Comment by pnrobinson** [comment]
@MickeySegal 
not an expert, but does this sound reasonable
new parent term
Megacolon
def: Persistent and substantial increase in diameter diameter and length of the colon.
comment: In addition to Hirschspring disease, acute colonic pseudoobstruction, idiopathic megacolon, severe chronic constipation, hypothyroidism, intestinal motility disorders, and toxic Megacolon may present with this feature.
-- and then in addition to the current term, have
Non-aganglionic megacolon (not sure if there is a better label)

**Comment by MickeySegal** [comment]
Sounds good.

**Comment by pnrobinson** [comment]
@MickeySegal  
HP:6000852
Megacolon
